### PR TITLE
Dynamic schema loading on startup from schema registry.

### PR DESCRIPTION
**Pull request by andricDu**
Schema loading from the registry works and I've verified in the debugger that the SchemaManager has the json config mentioned schema in its storage and that Generic Records can be created and properly validated.

As there is no "queued" queue, the params schema never exists in RabbitMQ so it is simply used for in-flux validation of input `Map<String,Object>` through a handle function. 

https://github.com/icgc-argo/workflow-graph-node/issues/15

### CODE REVIEW

**Comment by andricDu** [comment]
Merge conflicts! Drat!

**Comment by andricDu** [comment]
ok, ready now.

File: example-node-config.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
   "workflow": {
     "url": "icgc-argo/nextflow-data-processing-utility-tools",
     "revision": "2.4.0",
-    "schema": "org.icgc_argo.schemas.WorkflowParams",
-    "schemaAvscPath": "/avro/WorkflowParams.avsc"
+    "schemaName": "WorkflowParamsTest",
+    "schemaNamespace": "org.icgc_argo.workflow_graph_lib.schema",
```
Comment (andricDu) [inline example-node-config.json:37]: I should see if content type works with the full name of the schema, if it does, i might collapse this back to one field.

File: src/main/java/org/icgc_argo/workflowgraphnode/config/AppConfig.java [hunk 2]
```diff
@@ -1,27 +1,68 @@
...
-  public AppConfig(@Value("${node.jsonConfigPath}") String jsonConfigPath) {
+  public AppConfig(
+      @Value("${node.jsonConfigPath}") String jsonConfigPath,
+      @Autowired ReactiveRabbit reactiveRabbit,
+      @Autowired Environment environment) {
     val inputStream = new FileInputStream(jsonConfigPath);
     this.nodeProperties = new ObjectMapper().readValue(inputStream, NodeProperties.class);
+    this.reactiveRabbit = reactiveRabbit;
+
+    if (!asList(environment.getActiveProfiles()).contains("test")) {
```
Comment (andricDu) [inline src/main/java/org/icgc_argo/workflowgraphnode/config/AppConfig.java:40]: eventually need filesystem,classpath loading as well.

**Review by lepsalex — APPROVED** [review]

**Pull request merged**